client/asset/dcr/dcr.go

```diff
@@ -2932,7 +2849,7 @@ func (dcr *ExchangeWallet) signTxAndAddChange(baseTx *wire.MsgTx, feeRate uint64
 	// Sign the transaction to get an initial size estimate and calculate
 	// whether a change output would be dust.
 	sigCycles := 1
-	msgTx, err := dcr.signTx(baseTx)
+	msgTx, err := dcr.wallet.SignRawTransaction(dcr.ctx, baseTx.Copy())
```

Comment:
Maybe a comment here and other call sites why using `baseTx.Copy()` instead of just `baseTx`.